src/widgets/LayersAndLegends/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { LayersAndLegends } from './components/LayersAndLegends';
```

Comment:
it differs from our convention we have. 
let's see how views organized, we import ~views/BivariateManager/BivariateManager
it seems like  you need to LayersAndLegends/index.tsx -> LayersAndLegendsWidget/LayersAndLegendsWidget.tsx